src/mito2/src/engine.rs

```diff
@@ -112,13 +113,15 @@ impl MitoEngine {
         mut config: MitoConfig,
         log_store: Arc<S>,
         object_store_manager: ObjectStoreManagerRef,
+        kv_backend: KvBackendRef,
```

Comment:
better to use `SchemaMetadataManager`? cc @evenyag 
```suggestion
        schema_metadata_manager: SchemaMetadataManager
```

Reply:
fixed in https://github.com/GreptimeTeam/greptimedb/pull/4926/commits/6a82a59367c8960c81274a973f8d963b73b47b90